README.md

```diff
@@ -4,9 +4,77 @@
 [![docs.rs](https://docs.rs/finalfusion/badge.svg)](https://docs.rs/finalfusion/)
 [![Travis CI](https://img.shields.io/travis/finalfusion/finalfusion-rust.svg)](https://travis-ci.org/finalfusion/finalfusion-rust)
 
-This is a crate for reading, writing, and using finalfusion embeddings in
-Rust. Additionally, the word2vec and GloVe file formats are also supported.
-Please consult the [API documentation](http://docs.rs/finalfusion/) for
-usage information.
+`finalfusion` is a crate for reading, writing, and using embeddings in
+Rust. `finalfusion` primarily works with
+[its own format](https://finalfusion.github.io/spec) which supports a large
+variety of features. Additionally, the fastText, word2vec and GloVe file
+formats are also supported.
 
-**Note:** This package is still new, its API will change.
+## Usage
+
+To make `finalfusion` available in your crate, simply place the following
+in your `Cargo.toml`
+
+~~~
+finalfusion = 0.10
+~~~
+
+Loading embeddings and querying it is as simple as:
+
+~~~Rust
+use std::fs::File;
+use std::io::BufReader;
+
+import finalfusion::prelude::*;
+
+fn main() {
+    let mut reader = BufReader::new(File::open("embeddings.fifu").unwrap());
+    let embeds = Embeddings::<VocabWrap, StorageWrap>::read_embeddings(&mut reader).unwrap();
+    embeds.embedding("Query").unwrap();
+}
+~~~
+
+## Features
+
+`finalfusion` supports a variety of formats:
+
+* Vocabulary
+    * Subwords
+    * No subwords
+* Storage
+    * Array
+    * Memory-mapped
+    * Quantized
```

Comment:
Do we also need to mention memory-mapped quantized array here?

Reply:
I'm not sure, quantization and memory-mapping are describing different things. Memory-mapping just describes how the embedding storage can be accessed while array and quantized describe how they are stored in memory.

Maybe we can come up with a better over-all descriptions of the features? It's not really clear here how the three points can be combined, but it's what we use throughout the documentation (e.g. in `finalfusion-python` and on the webpage).